config/axe/axe.config.js

```diff
@@ -1,97 +0,0 @@
-/*jshint node: true*/
-'use strict';
-
-// Defaults derived from: https://github.com/dequelabs/axe-core
-const defaults = {
```

Comment:
Removing aXe stuff since e2e a11y testing is deprecated.
See: https://developer.blackbaud.com/skyux/learn/get-started/advanced/accessibility-analyzer
And: https://github.com/blackbaud/skyux-sdk-e2e/pull/24